perun-core/src/test/java/cz/metacentrum/perun/core/entry/SecurityTeamsManagerEntryIntegrationTest.java

```diff
@@ -347,23 +347,36 @@ public void testGetAdminsEmpty() throws Exception {
 		setUpUsers();
 
 		List<User> expected = Collections.emptyList();
-		List<User> actual = securityTeamsManagerEntry.getAdmins(sess, st0);
+		List<User> actual = securityTeamsManagerEntry.getAdmins(sess, st0,false);
 		Collections.sort(expected);
 		Collections.sort(actual);
 		assertEquals(expected, actual);
 	}
 
+	@Test
+	public void testGetAdminGroups() throws Exception {
+		System.out.println(CLASS_NAME + "testGetAdminGroups");
+		setUpSecurityTeams();
+		setUpUsers();
+		Group group = setUpGroup(u1, u2);
+		setUpAdmins(u0, u1, group);
+
+		//securityTeamsManagerEntry.addAdmin(sess, st0, group);
+
+		assertTrue(securityTeamsManagerEntry.getAdminGroups(sess, st0).contains(group));
+	}
+
```

Comment:
There is probably missing test for getAdmins(sess, securityTeam, true) which means only direct admins.